unittests/test_schedulers.py

```diff
@@ -743,6 +744,16 @@ def test_sched_access_idle(self):
         self.prepare_job()
         assert self.testjob.num_tasks == 8
 
+    def test_sched_access_idle_sequence_view(self):
```

Comment:
I don't think this is needed. Just change the `_sched_access` assignment in one of the `sched_access` tests.